admin/AmazonAI-Common.php

```diff
@@ -586,6 +591,34 @@ public function get_tim_limitless_viewkey(){
         return $value;
     }
 
+    public function get_languages_array_tim_limitless(){
+        $result = $this->curl_get_tim_limitless(TIM_LIMITLESS_LANGUAGES.$this->get_tim_limitless_installkey());
+        $tim_langs = json_decode($result);
+        $languages = array();
+        foreach ($tim_langs as $key => $lang){
+            $languages[] = ['code' => $key, 'name' => $lang->name, 'transable' => '1', 'polly' => '1'];
+        }
+        return $languages;
+    }
+
+    public function get_tim_limitless_posthash($post_id){
+        $postHash="";
+        if(!$this->is_excerpt_adder_enabled()){
+            if($this->is_title_adder_enabled()){
+                $postHash = get_post_meta( $post_id, TIM_LIMITLESS_POST_HASH_CONTENT_TITLE, true );
```

Comment:
second param is the only thing that changes
put it in variable and make one call to

get_post_meta( $post_id, $VARIABLE , true );